packages/framework/src/Model/ContactForm/ContactFormFacade.php

```diff
@@ -36,7 +36,7 @@ public function sendMail(ContactFormData $contactFormData)
             null,
             $this->getMailBody($contactFormData),
             t('Contact form'),
-            $contactFormData->email,
+            $this->mailSettingFacade->getMainAdminMail($this->domain->getId()),
             $contactFormData->name,
```

Comment:
question to @pk16011990 - what about the name - would not it make more sense to rather use `$this->mailSettingFacade->getMainAdminMailName($this->domain->getId());`  as "from name" here? :thinking:

Reply:
Good point, I agree. The sender name must be admin name, not user name 👍

Reply:
@vitek-rostislav @pk16011990 changed 🤝